app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class CommentsController < AuthenticationController
+  def destroy
+    policy.comment.destroy
+    render(json: { partial: comment_form })
```

Comment:
Json because you're using Javascript?

Reply:
Yes...although I have to admit I copied this from a pattern I'd used on another project. I think it probably makes sense though?

Reply:
Makes sense. destroy, assuming it works, can only go wrong with the destroy cascade. Otherwise it's easy street.